pkg/util/output.go

```diff
@@ -0,0 +1,72 @@
+package util
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/olekukonko/tablewriter"
+)
+
+// output style
+const (
+	OutputStylePlain = "plain"
+	OutputStyleTable = "table"
+	OutputStyleJson  = "json"
+)
+
+func RenderString(format string, headers []string, values [][]string) {
```

Comment:
@sillydong 

following the go style, can you add the comments for these public functions?